core/src/test/java/org/hibernate/ogm/backendtck/queries/QueriesWithEmbeddedCollectionTest.java

```diff
@@ -93,35 +96,40 @@ public void testEqualEmbeddedCollectionWithEmbeddableInCollectionWhereClause() t
 
 	@Test
 	public void testConjunctionOperatorWithEmbeddedInEmbeddedCollection() throws Exception {
-		List<?> result = session.createQuery( "FROM WithEmbedded e JOIN e.anEmbeddedCollection c WHERE c.item = 'item[1]' AND c.anotherItem IN ('secondItem[0]')" ).list();
+		List<?> result = session.createQuery( "FROM WithEmbedded e JOIN e.anEmbeddedCollection c WHERE c.item = 'item[0]' AND c.anotherItem IN ('secondItem[0]')" ).list();
 		assertThat( result ).onProperty( "id" ).containsOnly( 1L );
 	}
 
 	@Test
+	@SkipByGridDialect(value = GridDialectType.NEO4J)
```

Comment:
If it is relatively easily fixable prior to ORM 5, we can do it. I am
looking into this issue as we speak.